css/responsive/tablet.styl

```diff
@@ -47,7 +47,17 @@ body.single-collection
   flex-wrap: nowrap
 
 .single-collection-item
+  height: 100%
```

Comment:
this is causing distortion on my local. A landscape image is stretching to fill height. But maybe the images are always portrait?